PowerPointLabs/PowerPointLabs/PowerPointLabs.csproj

```diff
@@ -2073,6 +2073,9 @@
   <ItemGroup>
     <None Include="Resources\FillSlide.png" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="Resources\AddToCustomShapesDisabled.png" />
```

Comment:
Probably this should be None Include instead, which is causing the deployment error. Still testing.